### ISSUE TITLE
Crash with Fancy Menu and Ae2 Wireless Terminal

### ISSUE THREAD

**Issue by Saereth** [Mari023/AE2WirelessTerminalLibrary#243]
I'm running into a crash when using these two mods together and trying to open to wireless crafting terminal UI. I'm not sure on which mod's side the issue lies so I've reported it to both of you. If I can provide any addition information please let me know and I'll be happy to respond. 

**To Reproduce**
Steps to reproduce the behavior (e.g.):
Install a Minecraft 1.19.2 Minimal mod list

No customizations setup in Fancy Menu, all default settings
Setup a basic ae system with a wireless crafting terminal and try to open it. 
The game crashes with a divide by zero error. 

**Expected behavior**
For the screen to open and not crash

**Game Log**
Crash Log: https://gist.github.com/Saereth/8f65cdf663b2392fe41f49910e1fee60
Latest.log: https://gist.github.com/Saereth/a59eea5a7b46d54b136b23eb93f97baf

**Basic Informations (please complete the following information):**
 - OS: Windows 11 Pro
 - FancyMenu Version: 2.14.13
 - Forge/Fabric Version: 43.3.8
 - Minecraft Version: 1.19.2
 - Active Mods:

 ```
Forge 43.3.8
Ae2 12.9.9
Ae2 Wireless Terminal 12.9.7
JEI 
Jade
Curios Api
Konkrete 1.8.0
Auudio 1.0.3
Melody 1.0.1
Architectury 6.5.90
Fancy Menu 3.0.6_MC_1.19.2
```

Issue for Reference:
https://github.com/Keksuccino/FancyMenu/issues/698

**Comment by Mari023** [comment]
does this also happen with other ae2 guis / without ae2wtlib

**Comment by Saereth** [comment]
This does not occur with any of the standard AE2 Uis. Nor does the crash occur if Fancy menu is removed.

**Comment by Keksuccino** [comment]
Hey, dev of FancyMenu here! I can probably fix that on my side! :)

**Comment by Keksuccino** [comment]
This should be fixed in FancyMenu v3.1.0!
